File: Practicas/Marco-Oregel/Practica1/PayPalPayment.py
from PaymentMethod import PaymentMethod

class PayPalPayment(PaymentMethod):

    def __init__(self,email,password):
        self._email = email
        self._password = password

    def processPayment(self,amount):
        print(f"Se ha procesado su pago de: $ {amount} pesos con PayPal de la cuenta: {self._email}")

File: Practicas/Yahir-Hermosillo/Parcial3/Practica1/PayPalPayment.java
class PayPalPayment extends PaymentMethod {
    private String email;
    private String password;
    
    public PayPalPayment(String email, String password) {
        this.email = email;
        this.password = password;
    }
    
    @Override
    public void processPayment(double amount) {
        System.out.printf("Procesando pago con PayPal de $%.2f - Email: %s%n", amount, email);
    }
}